api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/I18nBundle.java

```diff
@@ -25,7 +25,7 @@
 	private static final Log log = LogFactory.getLog(I18nBundle.class);
 	private static final String startSep = "\u25a4";
 	private static final String endSep = "\u25a5";
-	private static final String intSep = "\u25a6";
+	public static final String intSep = "\u25a6";
```

Comment:
Personally, I prefer naming constants in accordance with Java naming conventions - https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html. Can you please apply that convention here? For instance, START_SEP.

Reply:
Sure

Reply:
Thanks